templates/partners/desktop.html

```diff
@@ -6,144 +6,243 @@
 
 {% block meta_description %}Canonical collaborates with the world&rsquo;s leading vendors to optimise and certify Ubuntu on 100s of laptops, desktops and workstations for a secure and constantly updated platform.{% endblock %}
 
+{% set is_paper = True %}
+
 {% block content %}
-  <section class="p-strip--suru-topped">
-    <div class="row">
-      <div class="col-7 col-medium-4">
-        <h1>Desktop Programme</h1>
-        <p>Canonical collaborates with the world&rsquo;s leading vendors to optimise and certify Ubuntu on 100s of laptops, desktops and workstations for a secure and constantly updated platform.</p>
-        <p>Whether built with the developer in mind or to power AI ambitions, these out-of the-box platforms give practitioners everything they need to transform their enterprise IT.</p>
-        <a href="https://certification.ubuntu.com/desktop" class="p-button"><span>Find a certified PC</span></a><a href="/partners/become-a-partner" class="p-button--positive">Become a partner</a>
-      </div>
-      <div class="col-5 col-medium-2 u-vertically-center">
-        <ul class="p-logo-section__items">
-          {% for partner in partners %}
-          <li class="p-logo-section__item">
-            <img class="p-logo-section__logo" src="{{ partner.logo }}" style="min-width: 5rem;" alt="{{ partner.slug}}">
-          </li>
-          {% endfor %}
-        </ul>
-      </div>
+
+<div class="row">
+  <div class="p-breadcrumbs u-sv1" aria-label="Breadcrumbs">
+    <ol class="p-breadcrumbs__items p-text--small-caps">
+      <li class="p-breadcrumbs__item "><a href="/partners">Partners</a></li>
+      <li class="p-breadcrumbs__item ">Desktop</li>
+    </ol>
+  </div>
+  <hr>
```

Comment:
all hrs in the new designs should be p-rules, unless there's a button immediately above them
```suggestion
  <hr class="p-rule">
```